src/KeyConnector/Services/CryptoFunctionService.cs

```diff
@@ -8,7 +8,7 @@ public class CryptoFunctionService : ICryptoFunctionService
     {
         public async Task<byte[]> AesGcmEncryptAsync(byte[] data, byte[] key)
         {
-            using var aes = new AesGcm(key);
+            using var aes = new AesGcm(key, AesGcm.TagByteSizes.MaxSize);
```

Comment:
ℹ️ Other deprecations. I _think_ this is the correct initialization size but needs review.